src/app/CMakeLists.txt

```diff
@@ -22,7 +22,11 @@
 # Setup main application
 ###########################################
 
-set(EXECUTABLE_NAME mscore)
+if (OS_IS_WASM)
+    set(EXECUTABLE_NAME MuseScoreStudio)
```

Comment:
Let's maybe do this below, under "platform specific", because for other platforms it is also done that way